sp800-63-3/sec1_2_introduction.md

```diff
@@ -133,10 +133,9 @@ operations of their digital systems.
 These technical guidelines do not address the
 authentication of a person who is physically present, for example, for
 access to buildings, even though some authenticators that are used
-remotely may also be used for local authentication. These guidelines do not yet address
-machine-to-machine (such as router-to-router) authentication, or
-establish specific requirements for issuing authenticators to machines and servers when they are used in
-authentication protocols with people.
+remotely may also be used for local authentication. In addition, these guidelines do not yet address
+machine-to-machine (such as router-to-router) authentication or specific guidance for authentication and authorization of interconnected devices, commonly referred to as the "internet of things (IoT)". Also excluded are specific requirements for issuing authenticators to devices when they are used in
+authentication protocols with people. Device identity is being considered for future revisions, which is why the title of this special publication was expanded to "Digital Identity Guidelines".
```

Comment:
Suggest moving the last sentence to the change history (section 2.5.3). Perhaps say something like "Publication title changed to "Digital Identity Guidelines" to accommodate possible future inclusion of device identity."